subiquity/common/types.py

```diff
@@ -371,6 +375,21 @@ class StorageResponseV2:
     install_minimum_size: Optional[int] = None
 
 
+class SizingPolicy(enum.Enum):
+    SCALED_FREE_SPACE = enum.auto()
+    ALL = enum.auto()
+
+    @classmethod
+    def from_string(cls, value):
+        if value is None:
+            return cls.SCALED_FREE_SPACE
+        if value == 'all':
+            return cls.ALL
+        if value in ('scaled-free-space', 'scaled'):
```

Comment:
any specific reasons for having two values here?

Reply:
no, I need to pick.  'scaled' it is.